docs/detections/api/exceptions/api-import-exception-list.asciidoc

```diff
@@ -0,0 +1,91 @@
+[[exceptions-api-import-exception-list]]
+=== Import exception list
+
+Imports an exception list and associated items.
+
+An exception list groups <<exceptions-api-create-exception-item, exception items>>
+and can be associated with rules. When an exception item's query evaluates to
+`true`, rules do *not* issue alerts even when the rule's other criteria are met.
+
+You can assign detection rules with multiple exception lists. For more information, refer to <<rules-api-create>> and <<rules-api-update>>.
```

Comment:
```suggestion
You can assign multiple exception lists to a detection rule. For more information, refer to <<rules-api-create>> and <<rules-api-update>>.
```

Reply:
```suggestion
You can assign multiple exception lists to a rule. For more information, refer to <<rules-api-create>> and <<rules-api-update>>.
```

Reply:
@benironside 's edits look good! I just left one more minor suggestion. Detection rules are generally referred to as just "rules" in the Security docs.